index.html

```diff
@@ -205,10 +205,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
 
       </div>
       <div class="form-inputs">
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/mlezozpr" method="post">
+        <input name="name" type="text" placeholder="Full name" maxlength="50" required>
+        <input name="email" type="email" placeholder="Email address" maxlength="50" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
+      </form>
       </div>
     </section>
```

Comment:
- [ ] Please adjust the `height`, the `border`, and the `background-color` of the `textarea `to match the Figma design attached below 👇 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166438126-b3efcbad-37f0-4893-b2ee-d3e6f3d8c531.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/166438173-21d58def-6cce-4762-bd36-bd0179d61143.png)